src/arch/x86/kernel/cmdline.c

```diff
@@ -136,6 +136,19 @@ void cmdline_parse(const char *cmdline, cmdline_opt_t *cmdline_opt)
         serial_init(cmdline_opt->console_port);
         x86KSconsolePort = cmdline_opt->console_port;
     }
+    printf("Console Com1 0x3f8\n");
+    serial_init(0x2f8);
+    x86KSconsolePort = 0x2f8;
+    printf("Console Com2 0x2f8\n");
+    serial_init(0x3e8);
+    x86KSconsolePort = 0x3e8;
+    printf("Console Com3 0x3e8\n");
+    serial_init(0x2e8);
+    x86KSconsolePort = 0x2e8;
+    printf("Console Com4 0x2e8\n");
+    serial_init(0x3f8);
+    x86KSconsolePort = 0x3f8;
+    
```

Comment:
There's an accidental change committed here it seems.

Reply:
Arrgh yes... just pushed a version without it.